WSL/index.md

```diff
@@ -1,37 +1,39 @@
 ---
 title: An overview on the the Windows Subsystem for Linux
-description: Learn about the Windows Subsystem for Linux, the different versions and ways you can use them.
+description: Explore the Windows Subsystem for Linux documentation.
```

Comment:
Was this file moved out of the TOC? I can't seem to access it anymore.

Reply:
Which file?